File: BackEnd/Core/Models/SearchRequest.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Core.Models
{
    public class SearchRequest
    {
        public int Epochs { get; set; }
        public int PopulationSize { get; set; }
        public double MutationProbability { get; set; }
    }
}

File: FrontEnd/ClientApp/src/app/models/search-request.ts
export class SearchRequest{
    public epochs : number;
    public populationSize : number;
    public mutationProbability : number;
    public constructor(epochs: number, populationSize : number, mutationProbability : number){
        this.epochs = epochs;
        this.populationSize = populationSize;
        this.mutationProbability = mutationProbability;
    }
}